E621Client/Converters/StringJsonConverter.cs

```diff
@@ -12,6 +12,8 @@ internal abstract class StringJsonConverter<T> : JsonConverter<T>
     {
         public sealed override T ReadJson(JsonReader reader, Type objectType, T existingValue, bool hasExistingValue, JsonSerializer serializer)
         {
+            if (reader.Value == null)
+                return ReadString(String.Empty, objectType, existingValue, hasExistingValue, serializer);
```

Comment:
I don't see what is happening here. Why are we passing in an empty string when the value is null?

If the reason is so that nulls and empty strings are treated the same, wouldn't it then be cleaner to check for IsNullOrWhiteSpace and then pass null in this class?